src/dll/BlenderPatchPatterns.h

```diff
@@ -346,6 +346,24 @@ static std::unordered_map<std::string, BlenderPatchPattern> Patchers{
             "E8 EB BD 00 00 48 85 C0",
         },
     },
+    {
+        "3.6.16",
+        {
+            "E8 EC B5 41 00",
+            "4C 8D 05 19 08 00 00",
+            "E8 7B FF FF FF 84 C0",
+            "E8 EB BD 00 00 48 85 C0",
+        },
+    },
+    {
+        "3.6.16",
```

Comment:
This line probably a typo?

```suggestion
        "3.6.17",
```